README.md

```diff
@@ -4,29 +4,37 @@ As of July 2017
 
 ## Table of Contents
 
-[League Play](#league-play)
-
-[Divisions](#divisions)
-
-[Tournaments](#tournaments)
+[Format](#format)
 
 [Matches](#matches)
 
+[Seasons](#seasons)
+
 [Teams](#teams)
 
-[Heroes](#heroes)
+## Format
 
-## League Play
+Teams play a best of 3 series against an evenly matched opponent each week on the chosen league night.
 
-### Weekly Matches
+### Scheduling
       
-Pick the available night of the week for your team to compete.  Whether you throw down on Thursday or take up arms on Tuesday, active teams will play a best of 3 series against an evenly matched opponent each week on the chosen night. Schedules come out the Friday before each week. Matches that are not completed on the website by the time the scheduler runs will be deleted, so make sure to post your results! A team can disable match scheduling at any time by [visiting your team's leagues page](https://www.chairleague.com/account/teams). Scheduling can be reenabled whenever your team is ready to play again.
+Match schedules are released the Friday before each week. A team can disable match scheduling at any time by [visiting your team's leagues page](https://www.chairleague.com/account/teams). Scheduling can be reenabled whenever your team is ready to play again.
+
+Be aware that **matches that are not recorded on the website by the time the scheduler runs will be deleted, so make sure to [post your results](#results)** promptly! Additionally, if there is an odd number of teams in a division on a certain week, one team will have a bye for that week.
+
+### Divisions
 
-### Competitive Map Pool
+Each league consists of several divisions with varying levels of competitiveness. Division 1 has the highest level of competition, followed by Division 2 and then Division 3. Teams may choose which division to compete in. Currently, there is no way to change divisions mid-season. Division choice should be based not on individual skill, but rather by strength as a team. That said, the rough breakdown of ranks in each division is:
 
-Chair League seeks to give you a fully immersive competitive experience.  To add to that only the most competitive maps are available for play, sorry Garden of Terror...  
+- Players in Division 1 tend to be Diamond or higher
+- Players in Division 2 tend to be between Gold and Diamond
+- Players in Division 3 tend to be between Bronze and Platinum
```

Comment:
Yea.  I may be overly afraid of 1st game shock from new to CL players now that it is pay 2 play.  Just want to take all steps necessary to prevent people feeling like they were misled by the descriptions given the number of teams with diamond (hotslog diamond that is so maybe just high plat in reality) players right after spending money.  If you are diamond and new maybe you join div3 to gauge where you stand amd the description will help push those players to div2.  I sse your point.  Probably better to see what the reception is after a season the way you have it than to change it now.